src/cfnlint/decode/cfn_json.py

```diff
@@ -537,16 +364,227 @@ class CfnJSONDecoder(json.JSONDecoder):
 
     def __init__(self, *args, **kwargs):
         json.JSONDecoder.__init__(self, *args, **kwargs)
-        self.parse_object = CfnJSONObject
+        self.parse_object = self.cfn_json_object
         self.parse_array = self.JSONArray
         self.parse_string = py_scanstring
         self.memo = {}
         self.object_pairs_hook = check_duplicates
         self.scan_once = py_make_scanner(self)
+        self.newline_indexes = []
+
+    def decode(self, s, _w=WHITESPACE.match):
+        """Overridden to retrieve indexes """
+        print(_w)
```

Comment:
Similar problem, super() wouldn't work right with python2, and the way the override worked pylint complained it did not have the same number of arguments.